src/base/torrentfileswatcher.cpp

```diff
@@ -320,7 +309,7 @@ void TorrentFilesWatcher::load()
     const QJsonObject jsonObj = jsonDoc.object();
     for (auto it = jsonObj.constBegin(); it != jsonObj.constEnd(); ++it)
     {
-        const QString &watchedFolder = it.key();
+        const auto &watchedFolder = Path(it.key());
```

Comment:
maybe?
```c++
const Path watchedFolder {it.key()};
```

And other similar instances in this file (where `auto` or `auto&` is used)

Reply:
Done.